test/react-native/TESTING.md

```diff
@@ -159,3 +159,9 @@ Remove
         <true/>
     </dict>
     ```
+
+1. Open `ios/reactnative.xcworkspace` in Xcode
+    1. Add a new Group called `Scenarios` beneath `reactnative/reactnative`
+    1. Add all files in `../../ios-modules/Scenarios` to the new group
+    1. Xcode may promt to add a bridging header for Swift.  Cancel this and instead:
```

Comment:
```suggestion
    1. Xcode may prompt to add a bridging header for Swift.  Cancel this and instead:
```

Reply:
Corrected on my next branch to avoid a whole new CI run.